internal/handlers/handlers.go

```diff
@@ -397,8 +398,9 @@ func ChannelsHandler(c *fiber.Ctx) error {
 		// Set the Content-Disposition header for file download
 		c.Set("Content-Disposition", "attachment; filename=jiotv_playlist.m3u")
 		c.Set("Content-Type", "application/vnd.apple.mpegurl") // Set the video M3U MIME type
-		c.Response().Header.Set("Cache-Control", "public, max-age=3600")
-		return c.SendString(m3uContent)
+		c.SendStream(strings.NewReader(m3uContent))
+		return nil
```

Comment:
```suggestion
		return c.SendStream(strings.NewReader(m3uContent))
```

Reply:
@dheerajv4855 If c.SendStream returns error, then we should go with this approach. Can you please check?

Reply:
yes got your point 
you are correct 
we usualy SendStream multiple times in for loop so there we do explicitly check err returned by SendStream 
in this case you are right we should return SendStream directly